### PR TITLE
[Editor] Fix shortcut reset.

### PR DESCRIPTION
**Pull request by bruvzg**
Fixes regression from https://github.com/godotengine/godot/pull/73422

See https://github.com/godotengine/godot/pull/73422#issuecomment-1437634414

### CODE REVIEW

**Review by akien-mga — APPROVED** [review]
Merging proactively to fix the regression for 4.0 RC3, but I'm not 100% if this works as I'd expect it in all cases.

Notably say for an editor plugin that would register a new editor setting with `add_shortcut` (which should be considered the original, but not customized), and which can then be customized by the user of that plugin.

**Comment by akien-mga** [comment]
Thanks!

**Comment by EricEzaM** [comment]
@akien-mga Do you mean from script? `add_shortcut` is not exposed to scripting currently.

File: editor/editor_settings.h [hunk 1]
```diff
@@ -170,6 +170,7 @@ class EditorSettings : public Resource {
 	String get_editor_layouts_config() const;
 	float get_auto_display_scale() const;
 
+	void _add_shortcut_default(const String &p_name, const Ref<Shortcut> &p_shortcut);
```
Comment (EricEzaM) [inline editor/editor_settings.h:173]: Doesn't need to be public.

**Pull request merged**